src/datasource.ts

```diff
@@ -475,75 +482,27 @@ export class QuickwitDataSource
     return text;
   }
 
-  private makeLogContextDataRequest = (row: LogRowModel, options?: LogRowContextOptions) => {
-    const direction = options?.direction || LogRowContextQueryDirection.Backward;
-    const searchAfter = row.dataFrame.fields.find((f) => f.name === 'sort')?.values.get(row.rowIndex) ?? [row.timeEpochNs]
-
-    const logQuery: Logs = {
-      type: 'logs',
-      id: '1',
-      settings: {
-        limit: options?.limit ? options?.limit.toString() : '10',
-        // Sorting of results in the context query
-        sortDirection: direction === LogRowContextQueryDirection.Backward ? 'desc' : 'asc',
-        // Used to get the next log lines before/after the current log line using sort field of selected log line
-        searchAfter: searchAfter,
-      },
-    };
-
-    const query: ElasticsearchQuery = {
-      refId: `log-context-${row.dataFrame.refId}-${direction}`,
-      metrics: [logQuery],
-      query: '',
-    };
-
-    const timeRange = createContextTimeRange(row.timeEpochMs, direction);
-    const range = {
-      from: timeRange.from,
-      to: timeRange.to,
-      raw: timeRange,
-    };
-
-    const interval = rangeUtil.calculateInterval(range, 1);
-
-    const contextRequest: DataQueryRequest<ElasticsearchQuery> = {
-      requestId: `log-context-request-${row.dataFrame.refId}-${options?.direction}`,
-      targets: [query],
-      interval: interval.interval,
-      intervalMs: interval.intervalMs,
-      range,
-      scopedVars: {},
-      timezone: 'UTC',
-      app: CoreApp.Explore,
-      startTime: Date.now(),
-      hideFromInspector: true,
-    };
-    return contextRequest;
-  };
-
-  getLogRowContext = async (row: LogRowModel, options?: LogRowContextOptions): Promise<{ data: DataFrame[] }> => {
-    const contextRequest = this.makeLogContextDataRequest(row, options);
-
-    return lastValueFrom(
-      this.query(contextRequest).pipe(
-        catchError((err) => {
-          const error: DataQueryError = {
-            message: 'Error during context query. Please check JS console logs.',
-            status: err.status,
-            statusText: err.message,
-          };
-          throw error;
-        })
-      )
-    );
-  };
+  // Log Context
 
+  // NOTE : deprecated since grafana-data 10.3
```

Comment:
can you open an issue on this? we should also upgrade all packages to 10.3 and docker image too.